src/stream/src/executor/managed_state/join/mod.rs

```diff
@@ -141,6 +141,24 @@ impl EncodedJoinRow {
         self.degree -= 1;
         Ok(self.degree)
     }
+
+    // TODO(yuhao): only need to decode part of the encoded row.
+    // TODO(yuhao): add kv api in state table to avoid manually append pk prefix.
+    /// Get a row with the schema in degree state table
+    ///
+    /// * `state_order_key_indices` - the order key of `row`
+    pub fn get_schemaed_degree(
+        &self,
+        row_data_types: &[DataType],
+        state_order_key_indices: &[usize],
+    ) -> StreamExecutorResult<Row> {
+        let degree_datum = Some(ScalarImpl::Int64(self.degree as i64));
+        let prefix = self
+            .decode_row(row_data_types)?
+            .by_indices(state_order_key_indices);
+        let schemaed_degree = state_table::append_pk_prefix(Row::new(vec![degree_datum]), prefix);
+        Ok(schemaed_degree)
+    }
```

Comment:
The degree table schema is pk of table + degree. To fit the state table API, I have to manually add the pk to the prefix of degree. We may need a kv API for state table